clib.json

```diff
@@ -4,5 +4,6 @@
   "repo": "Immediate-Mode-UI/Nuklear",
   "description": "A small ANSI C gui toolkit",
   "keywords": ["gl", "ui", "toolkit"],
+  "license": "MIT",
```

Comment:
``` suggestion
  "license": "MIT, Unlicense",
```
Looks like it just loads a string, so I suppose we can stick any comma-seperated SPDX text in here?
https://github.com/clibs/clib/blob/master/src/common/clib-package.c#L481